docker-build.sh

```diff
@@ -128,6 +128,22 @@ prepare_extra_common() {
     echo "fdk-aac-stripped${TARGET_DIR}/lib/libfdk-aac.so* usr/lib/jellyfin-ffmpeg/lib" >> ${DPKG_INSTALL_LIST}
     popd
     popd
+
+    # RKMPP
```

Comment:
These should not in `prepare_extra_common()`,  you can create an extra `prepare_extra_arm()` and call it at here https://github.com/jellyfin/jellyfin-ffmpeg/blob/816b853e99720165e37e01e2cae940c9d192070f/docker-build.sh#L520-L521.

Reply:
Done.